.github/archivarius.yml

```diff
@@ -0,0 +1,9 @@
+# Comment to post when potential duplicates are detected
+referenceComment: |
+  Thank you for the bug report :+1:
+  
+  Sadly, it appears some important information is still missing, which you may have overlooked while submitting the bug.
+  Without this information we may not be able to assistance you in resolving the issue.
```

Comment:
Me again: My favourite dictionaries know "assistance" only as noun. They suggest either "to assist" or "to give / provide / render assistance".

Reply:
> Me again: My favourite dictionaries know "assistance" only as noun. They suggest either "to assist" or "to give / provide / render assistance".

You're a great help, mate!
I've been agonising over this message and iterate a number of different variants. I completely missed this 🤦‍♂